newrelic.yml

```diff
@@ -372,6 +372,10 @@ common: &default_settings
   # prepend, chain, disabled.
   # instrumentation.bunny: auto
 
+  # Controls auto-instrumentation of roda at start up.
+  # May be one of [auto|prepend|chain|disabled]
+  # instrumentation.roda: auto
```

Comment:
Should we still manually add things to this file? I would have expected the change to happen via our CI.

Reply:
This was auto-added from the thor generator. I've deleted it and will let the CI handle it at a later date ◡̈